include/flash_map_pm.h

```diff
@@ -51,4 +47,11 @@
 #define FLASH_AREA_LABEL_EXISTS(label) \
 	IS_ENABLED(PM_IS_ENABLED(label))
 
+
```

Comment:
I'm no sure if no better to remove this check completely.

Reply:
I would say that we can remove this check since there the build system ensures that this partition is added. `littlefs_storage` is added as long as CONFIG_FILE_SYSTEM_LITTLEFS is set, regardless of whether or not `SETTINGS_FCB` or `SETTINGS_NVS` is set.